README.md

```diff
@@ -116,6 +116,16 @@ In order to start up oneview-redfish-toolkit service, there is some mandatory co
   
   * **DeliveryRetryIntervalSeconds**: The value of this property shall be the interval in seconds between the retry attempts for any given event to the subscription destination.
 
+* `scmb` section
+    
+  * **CAFile**: The CA certificate file for SCMB.
+
+  * **SCMBCertFile**: The SCMB cert file.
+
+  * **SCMBKeyFile**: The SCMB key file.
```

Comment:
maybe we could specify adding that these three properties is about the "file path"